Simple Featuretools RUL Demo.ipynb

```diff
@@ -355,83 +412,121 @@
-       "      <td>166</td>\n",
+       "      <td>2000-01-06 00:00:00</td>\n",
+       "      <td>206</td>\n",
        "    </tr>\n",
        "    <tr>\n",
-       "      <th>4</th>\n",
+       "      <th>3</th>\n",
        "      <td>4</td>\n",
-       "      <td>2000-01-08 01:30:00</td>\n",
-       "      <td>183</td>\n",
+       "      <td>2000-01-08 03:10:00</td>\n",
+       "      <td>173</td>\n",
        "    </tr>\n",
        "    <tr>\n",
-       "      <th>5</th>\n",
+       "      <th>4</th>\n",
        "      <td>5</td>\n",
-       "      <td>2000-01-10 15:10:00</td>\n",
-       "      <td>6</td>\n",
+       "      <td>2000-01-10 00:50:00</td>\n",
+       "      <td>92</td>\n",
        "    </tr>\n",
        "  </tbody>\n",
        "</table>\n",
        "</div>"
       ],
       "text/plain": [
-       "       engine_no         cutoff_time  RUL\n",
-       "index                                    \n",
-       "1              1 2000-01-03 00:00:00   32\n",
-       "2              2 2000-01-03 13:50:00  248\n",
-       "3              3 2000-01-06 06:40:00  166\n",
-       "4              4 2000-01-08 01:30:00  183\n",
-       "5              5 2000-01-10 15:10:00    6"
+       "    engine_no         cutoff_time  remaining_useful_life\n",
+       "id                                                      \n",
+       "0           1 2000-01-01 16:40:00                    220\n",
+       "1           2 2000-01-03 22:10:00                    198\n",
+       "2           3 2000-01-06 00:00:00                    206\n",
+       "3           4 2000-01-08 03:10:00                    173\n",
+       "4           5 2000-01-10 00:50:00                     92"
       ]
      },
-     "execution_count": 2,
+     "execution_count": 5,
      "metadata": {},
      "output_type": "execute_result"
     }
    ],
    "source": [
-    "cutoff_times = utils.make_cutoff_times(data)\n",
+    "label_times = lm.search(\n",
+    "    data.sort_values('time'),\n",
+    "    num_examples_per_instance=1,\n",
+    "    minimum_data=100,\n",
+    "    verbose=True,\n",
+    ")\n",
     "\n",
-    "cutoff_times.head()"
+    "label_times.head()"
    ]
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lines_to_next_cell": 2
+   },
    "source": [
-    "Let's walk through a row of the `cutoff_times` dataframe. In the third row, we have engine number 3. At 3:20 on January 6, the remaining useful life of engine number 3 is 213. Having a dataframe in this format tells Featuretools that the feature vector for engine number 3 should only be calculated with data from before that point in time. \n",
+    "Let's walk through a row of the `labels_times` dataframe. In the third row, we have engine number 3. At 00:00 on January 6, the remaining useful life of engine number 3 is 206. Having a dataframe in this format tells Featuretools that the feature vector for engine number 3 should only be calculated with data from before that point in time.\n",
```

Comment:
Is the engine number not deterministic?  In the old example engine 3 has 213 RUL several hours after engine 3 in the updated notebook has 206 RUL.

Reply:
Correct, the previous example didn't have deterministic label times. We also wanted to adjust the prediction problem to resemble the example in Compose.